internal/config/config.go

```diff
@@ -0,0 +1,171 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+// The config package handles all of the node's configurations,
+// which are mostly obtained through environment variables.
+//
+// The get.go file contains most of the getter functions for the configuration values.
+// Its code is automatically generated by the generate/main.go script
+// given the environment variable configured in the generate/Config.toml file.
+//
+// Custom getters are defined in the config.go file.
+package config
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"sync"
+	"time"
+)
+
+//go:generate go run generate/main.go
+
+// ------------------------------------------------------------------------------------------------
+// Parsing functions
+// ------------------------------------------------------------------------------------------------
+
+func toInt64FromString(s string) (int64, error) {
+	return strconv.ParseInt(s, 10, 64)
+}
+
+func toStringFromString(s string) (string, error) {
```

Comment:
```suggestion
func forwardString(s string) (string, error) {
```
Or something like that.